extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigDescriptionsManager.java

```diff
@@ -28,6 +28,9 @@ public class ConfigDescriptionsManager extends DevConsolePostHandler implements
     private final Set<String> devServicesProperties;
     private ClassLoader currentCl;
 
+    private static final char[] quotedDot = { '\"', '.', '\"' };
```

Comment:
Why not a String since you're `String.valueOf` below?

Reply:
Yea fair. Let me change that

Reply:
Done